File: JavaScript/level1/하샤드_수.js
function solution(x) {
    let temp = x;
    let sum = 0;
    
    while(temp != 0){
        sum += temp%10;
        temp = (temp - temp%10)/10;
    }
    
    if(x%sum === 0)
        return true;
    return false;
}

File: C++/level1/하샤드_수.cpp
#include <string>
#include <vector>

using namespace std;

bool solution(int x) {
    int temp = x;
    int sum = 0;
    
    while(temp != 0){
        sum += temp%10;
        temp /= 10;   
    }
    
    if(x%sum == 0)
        return true;
    return false;
}
